### DISCUSSION TITLE
Meeting Notes 16 July Kampus Hours

### DISCUSSION THREAD

**Discussion by Ketcap** [kamp-us/pano#518] [General]
Let's prepare notes, questions and anything that is related to it.

**Comment by cansirin** [comment]
Is there anything required for this day that I can prepare for better/smooth Kampus Hours?

**Reply by Ketcap** [reply]
No thanks @cansirin , I'll be doing open office hours either I'll answer some questions or I'll be working together with anybody that wants to join. In the meantime I'm open for questions challenges and assisting anyone that need.

**Comment by Ketcap** [comment]
Topics that are covered.

1. Pasaport App Specs Created and prepared https://github.com/kamp-us/monorepo/discussions/526
2. Auto-collapse generated files https://github.com/kamp-us/monorepo/pull/523
3. PR builds are fixed https://github.com/kamp-us/monorepo/pull/524, adjustment have been done in Vercel as well.(change to default build)
4. For the pasaport, base pages has been adjusted https://github.com/kamp-us/monorepo/pull/531
5. Covered topic about how to do contribution, status of the project and how the kampus will work
